src/constants.js

```diff
@@ -0,0 +1 @@
+export const defaultArea = 'default';
```

Comment:
If this is something internal let's place and ID that could not be duplicated by mistake, somethiing like _default-area_

Reply:
Ok, thank you

Reply:
Done